records_mover/db/redshift/redshift_db_driver.py

```diff
@@ -41,6 +42,7 @@ def __init__(self,
                              table=self.table,
                              temporary_s3_directory_loc=self.temporary_s3_directory_loc)
 
+    # TODO: this should be in loader class
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker